### PR TITLE
Change select to use activeIndex instead of :hover

### PR DESCRIPTION
**Pull request by jmophoto**
Change the select to use activeIndex to determine hover state instead of psuedoclass.

@bsbeeks @cerinman

### CODE REVIEW

**Comment by jmophoto** [comment]
This should also have a hover color or style. Adding that now.

**Comment by jmophoto** [comment]
Default:
screen shot 2015-10-20 at 1 21 01 pm

Passing red background color and green color:
```javascript
{{ backgroundColor: 'red', color: 'green' }}
```
screen shot 2015-10-20 at 1 20 50 pm

File: src/components/Select.js [hunk 1]
```diff
@@ -11,6 +11,7 @@ class Select extends React.Component {
   constructor (props) {
     super(props);
     this.state = {
+      activeIndex: this.props.options.indexOf(this.props.selected),
```
Comment (bsbeeks) [inline src/components/Select.js]: I don't think we'd want to set this by default. The hover styles should only display if you are actually hovering over an item.

**Comment by jmophoto** [comment]
- Set initial state of activeIndex to -1
- Set activeIndex to -1 on click
- Set activeIndex to -1 onMouseLeave

**Comment by bsbeeks** [comment]
There has to be a better way to solve this. Let's spend some more time on `.blur()` and maybe using different mouse events.

**Comment by bsbeeks** [comment]
@jmophoto where are we at with this?

**Comment by jmophoto** [comment]
@bsbeeks: will revive this!

**Comment by bsbeeks** [comment]
@jmophoto let's wrap this up or close it.

**Comment by lukeschunk** [comment]
my recent PR might address this - see what you guys think

**Comment by bsbeeks** [comment]
@lukeschunk agreed. Closing this in favor of your PR.

**Pull request closed without merging**